client/src/pages/RoomPage/index.tsx

```diff
@@ -30,6 +29,7 @@ function RoomPage({ name }: { name: string }): JSX.Element {
   const [selfUserInfo, setSelfUserInfo] = useState<UserInfo>({ ...defaultUserInfo, name });
```

Comment:
Since self user info still exists, might want to edit the title to be more clear?

Reply:
Changed title